JetNews/app/src/main/java/com/example/jetnews/ui/home/HomeScreen.kt

```diff
@@ -133,6 +133,8 @@ fun HomeScreen(
     navigateTo: (Screen) -> Unit,
     scaffoldState: ScaffoldState
 ) {
+    val scope = rememberCoroutineScope()
```

Comment:
Nit: Move this closer to the call site (i.e. `Scaffold`) and rename to `coroutineScope`